index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="./styles/defaultStyle.css">
+	<link rel="stylesheet" href="./styles/generalStyle.css">
+	<link rel="stylesheet" href="./styles/style.css">
+	<title>DiegoPortfolio</title>
+</head>
+
+<body>
+	<div class="firstSection">
+		<nav>
+			<ul class="toolbar">
+				<li>John Doe</li>
```

Comment:
- [ ] I think "John Doe" is the logo of your website 🚨, the logo should have a link (`a` tag). This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info. Kindly, add a `<a>` to your logo, and also you could customize it changing "John Doe" to your name. 💡